app/views/pages/_comments_card.html.erb

```diff
@@ -1,9 +1,13 @@
-<div class="container mt-2">
+<div class="container-fluid mt-2">
 <div class="card border-2 border-dark">
   <div class="card-body">
-    <% comments.each do |comment| %>
+  <% comments.each do |comment| %>
+    <% if comment.author %>
       <p><%= comment.author.name %>: <%= comment.text %></p>
+    <% else %>
+      <p><%= comment.text %></p>
     <% end %>
-  </div>
+  <% end %>
+</div>
```

Comment:
- [ ] As per project requirements, you are supposed to have a form in `views/comments/new.html.erb` file which accepts input for creating comments. This is currently missing. Kindly add it